server/graphql/common/expenses.ts

```diff
@@ -736,7 +725,7 @@ async function payExpenseWithPayPal(remoteUser, expense, host, paymentMethod, to
       fees['hostFeeInHostCurrency'],
       fees['platformFeeInHostCurrency'],
     );
-    expense.setPaid(remoteUser.id);
+    await markExpenseAsPaid(expense, remoteUser);
```

Comment:
This is what fixes https://opencollective.freshdesk.com/a/tickets/13223: `markExpenseAsPaid` also creates the activity.